pkg/util/fixtest/remove_duplicates.go

```diff
@@ -0,0 +1,65 @@
+package fixtest
+
+import (
+	"context"
+	"log"
+	"sort"
+
+	"go.mongodb.org/atlas/mongodbatlas"
+)
+
+type Logger interface {
```

Comment:
While I understand this is a utility function, why we can't simply use Zap instead of defining a new "random" interface?

Reply:
ok will make it zap aware and remove this interface

Reply:
Done, we are now using a `*zap.SugaredLogger` here instead.

I am not sure I like it. The test using this code would want to dump to a GinkgoWriter, and connecting the zap logger to such writer was not obvious at all.